content/foundations/getting-started.mdx

```diff
@@ -0,0 +1,74 @@
+
+Nielsen Norman Group [defines Responsive Web Design](https://www.nngroup.com/articles/responsive-web-design-definition/) as “a web development approach that creates dynamic changes to the appearance of a website, depending on the screen size and orientation of the device being used to view it”.
+
+“Adaptive design”, on the other hand, has been a term historically used to identify an approach where entirely separated fixed layouts were created to support mobile devices (MDN has good documentation [on how we got here](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Responsive_Design#historic_website_layouts)).
+
+“Responsive design” at GitHub is inherently adaptive, in the sense that it should not only adjust layout and spacing when resizing a page, but support patterns and component variants that match the paradigms and affordances of the person’s device:
+
+- Responsive to the form factor: adapt to viewport size, and pointing device support (`fine` for desktop, or `coarse`, for touch), and to the device metaphors and affordances.
+- Responsive to the user preferences: respect browser’s default font size, reduced motion, color scheme, contrast preferences, etc.
+
+Primer plays an important role in providing reusable patterns and components that can streamline the creation of responsive-friendly and accessible designs, letting teams focus on bigger problems. GitHub projects should take advantage of these patterns so they can provide people with cohesive experiences throughout the entire platform.
+
+Moving forward, “Responsive design” is the unified term used when discussing any design decisions related to device form factor and user preferences.
+
+## Design for efficiency
+
+GitHub is a platform used for productivity. We aim to encourage flow, focus, and an experience that is fast and compact.
+
+By providing a cohesive, inclusive, and responsive design, Primer’s goal is to remove as much friction as possible between the human and the software.
+
+Primer patterns and components aim towards accessible solutions that don’t dilute efficient UI interactions. It focuses on responsive design that doesn’t dilute desktop experiences into mobile-first solutions. Ultimately, it recognizes the need to encourage flow by providing calm and logical experiences throughout.
+
+## The zen of GitHub
+
+The [GitHub API](https://api.github.com/zen) consolidates the process and style in which GitHub makes design decisions in 14 aphorisms.
+
+More details about the zen of GitHub can be found in [@kneath](https://github.com/kneath)’s [post](https://warpspire.com/posts/taste).
+
+- Responsive is better than fast
```

Comment:
I somehow feel it would be nice to add a bit more context to those. Can't we add a "one sentence why" to every one? Or what is the effect of something e.g. `Encourage flow`. Otherwise it feels a bit excluding and top-down.

Reply:
That's a great idea. I think we should have "The Zen of GitHub" as a dedicated page in the future.